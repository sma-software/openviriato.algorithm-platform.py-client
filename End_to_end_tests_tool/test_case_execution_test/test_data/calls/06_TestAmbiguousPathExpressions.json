{
    "restResource": "/section-tracks",
    "httpMethod": "GET",
    "urlParameters": null,
    "queryParameters": [
        {
            "key": "ToNodeID",
            "value": "348",
            "optional": true
        },
        {
            "key": "FromNodeID",
            "value": "348",
            "optional": true
        }
    ],
    "requestBody": null,
    "CSharpMethodName": "GetSectionTracksTo(toNodeID)",
    "CSharpMethodParameterTypes": [ "System.Int64" ],
    "subcategory": "Get Section Tracks For Node"
}
