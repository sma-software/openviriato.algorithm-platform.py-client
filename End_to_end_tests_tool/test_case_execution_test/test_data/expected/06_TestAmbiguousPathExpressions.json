{
  "ID": 1588,
  "Code": "9012",
  "SectionCode": "2334",
  "Weight": 9,
  "DebugString": "DebugString"
}